src/application/api/open-api/responseHelpers.ts

```diff
@@ -101,8 +133,23 @@ const transformValidator = (validator: ValidatorRaw) =>
     }),
   )
 
+const transformToken =
+  (token: Token) => (values: (ResourceIdentifierT | AmountT)[]) => ({
+    name: token.token_properties.name ?? '',
+    rri: values[0] as ResourceIdentifierT,
+    symbol: token.token_properties.symbol,
+    description: token.token_properties.description,
+    granularity: values[1] as AmountT,
+    isSupplyMutable: token.token_properties.is_supply_mutable,
+    currentSupply: values[2] as AmountT,
+    tokenInfoURL: responseHelper.transformUrl(token.token_properties.url),
+    iconURL: responseHelper.transformUrl(token.token_properties.icon_url),
+  })
+
 export const responseHelper = {
```

Comment:
maybe we can just remove this object and just export the functions directly?

Reply:
fine by me. will change it